### DISCUSSION TITLE
Task Lab 03-02

### DISCUSSION THREAD

**Discussion by maomiranda30** [rramosp/ai4eng.v1#37] [03 - GENERACION DE MODELOS]
Buenas noche profe, tengo una duda, utilizando el bucle for yo puedo iterar varias listas y guardarlas todas ellas en una nueva lista con la funcion .append, pero si estoy utilizando el bucle for para crear distintos dataframes como los puedo iterar para que CADA  dataframe se guarde en un nuevo por ejemplo:

for i in list(range(0,4)):
     df.format(i)=d.shift(Period=i)

El anterior codigo no me da resultado , asi que no se si pueda responder mi duda o si me hice entender. 

Saludos :)

**Comment by Felipe-RA** [comment]
Saludos Maomiranda

No entiendo muy bien tu pregunta, ¿quieres crear un dataframe que tenga dataframes en su interior?

Si lo que intentas hacer es crear el dataframe con los `lookback`  de las señales  te recomiendo que mires hacer un ciclo  según  el número de loopbacks requeridos.  Te recomiendo que te mires la documentación de [pd.shift](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.update.html) y [pd.join ](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.join.html?highlight=join#pandas.DataFrame.join)

En especial los parámetros  `periods` y `rsufix`

**Reply by maomiranda30** [reply]
Listo les daré una lectura,  gracias por responder.
